### PR TITLE
Updating functions, fixing typos~

### PR DESCRIPTION
**Pull request by Neodevils**

### CODE REVIEW

**Comment by Neodevils** [comment]
@USERSATOSHI & @Leref, prioritized the functions needed to update. Ready to merge.

**Comment by Neodevils** [comment]
Will be updating summary.md along with array functions

**Comment by Neodevils** [comment]
@Leref

File: src/functions/usdisautocomplete.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+description: Returns boolean of slash choice.
+---
+
+# $isAutoComplete
+
+`$isAutoComplete` is used to check it is auto-complete or not.
```
Comment (USERSATOSHI) [inline src/functions/usdisautocomplete.md]: not specific enough here

**Review by USERSATOSHI — CHANGES_REQUESTED** [review]

File: src/functions/usdisautocomplete.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: Returns boolean of slash choice.
 
 # $isAutoComplete
 
-`$isAutoComplete` is used to check it is auto-complete or not.
+`$isAutoComplete` is used to checking the if interaction command's *interaction autocomplete* feature enabled or disabled.
```
Comment (USERSATOSHI) [inline src/functions/usdisautocomplete.md]: make it 
> check if interaction command is an *autocompleteInteraction*

**Review by USERSATOSHI — CHANGES_REQUESTED** [review]

**Review by Leref — APPROVED** [review]

**Pull request merged**